pkg/otelcgen/services.go

```diff
@@ -7,7 +7,7 @@ import (
 	"go.opentelemetry.io/collector/pipeline"
 	"go.opentelemetry.io/collector/service/extensions"
 	"go.opentelemetry.io/collector/service/pipelines"
-	"go.opentelemetry.io/collector/service/telemetry"
+	"go.opentelemetry.io/collector/service/telemetry/otelconftelemetry"
```

Comment:
they moved things around between versions https://pkg.go.dev/go.opentelemetry.io/collector/service/telemetry/otelconftelemetry#Config

Reply:
@0sewa0 https://github.com/open-telemetry/opentelemetry-collector/pull/13560 so we are good with change.